projects/cloud/app/graphql/mutations/change_user_role.rb

```diff
@@ -0,0 +1,14 @@
+module Mutations
+  class ChangeUserRole < ::Mutations::BaseMutation
+    argument :user_id, ID, required: true
+    argument :organization_id, String, required: true
+    argument :current_role, Types::RoleType, required: true
+    argument :new_role, Types::RoleType, required: true
```

Comment:
If we remove `current_role`, we can rename this one to `role` because it's obvious from the name of the mutation that the role is the one you are trying to set.